scalafmt-tests/src/test/scala/org/scalafmt/cli/CliTest.scala

```diff
@@ -1030,3 +1030,103 @@ class CliTest extends AbstractCliTest with CliTestBehavior {
   }
 
 }
+
+class CliTestSimp extends AbstractCliTest {
```

Comment:
I'm not familiar with the full test setup of this project.
These tests pass over here, but fail if pulled into the class above.

Reply:
Figured this out